maven-java/akkaserverless-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/pom.xml

```diff
@@ -299,18 +297,6 @@
     </profile>
   </profiles>
 
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>com.typesafe.akka</groupId>
-        <artifactId>akka-bom_${scala.binary.version}</artifactId>
-        <version>${akka-streams.version}</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
```

Comment:
don't we want to promote this anymore?

Reply:
We didn't have it in any of our samples. I don't know when or why it was introduced? It should work with transitive dependencies from sdk as long as user isn't starting to add Akka dependencies to their build.

Reply:
> I don't know when or why it was introduced?

Not sure either, looks like it's been there for ages (be8f8d6ea)

> It should work with transitive dependencies from sdk as long as user isn't starting to add Akka dependencies to their build.

This archetype *is* adding an Akka dependency to the build, though (akka-stream) - perhaps that can be removed?

Reply:
This *is* the archetype and and I'm removing the akka-stream dependency here.

Reply:
> I'm removing the akka-stream dependency here.

Aaah gotcha I didn't put those together yet :+1: